### ISSUE TITLE
Force cloudfront CDN

### ISSUE THREAD

**Issue by albertlarsan68** [rust-lang/cargo-bisect-rustc#282]
Fastly CDN is really slow (1MiB/s) when downloading data that is not in cache. Since this downloads specific nightlies that may be months old, there is quite a good chance that the downloads aren't in Fastly's cache.

I would think there would be a way to force the usage of CloudFront?

**Comment by Urgau** [comment]
Wasn't this done in https://github.com/rust-lang/cargo-bisect-rustc/commit/e5925b51edf4c66897b15219dab8a941272ddf85 ?

**Comment by albertlarsan68** [comment]
Nope, this changes the CI artifacts server from direct S3 to CDN, not the nightlies one from {Cloudfront,Fastly} to only Cloudfront.
